resources/user-guides/attach-data.md

```diff
@@ -110,25 +110,30 @@ After you complete your configuration, verify that your rule looks like the foll
 2. Rebuild your Launch property and deploy it to the correct Environment.
 {% endtab %}
 
-{% tab title="Target" %}
-### Attaching additional Target request objects to Target to retrieve location events
+{% tab title="Target" %} <a name="Target-Attach-Data-Tab"></a>
+
+{% hint style="warning" %}
+Attach data events apply to the Target Extension only. They do not apply to the Target VEC Extension. 
```

Comment:
Please reword as follows:
"Attach the data events that apply only to the Target extension. The data events do not apply to the Target VEC extension."